templates/default/slurm.conf.erb

```diff
@@ -8,50 +8,29 @@
 #
 # See the slurm.conf man page for more information.
 #
+# CLUSTER SETTINGS
 ClusterName=parallelcluster
-ControlMachine=<%= node['hostname'] %>
-#ControlAddr=
-#BackupController=
-#BackupAddr=
-#
 SlurmUser=slurm
-#SlurmdUser=root
 SlurmctldPort=6817
 SlurmdPort=6818
 AuthType=auth/munge
-#JobCredentialPrivateKey=
-#JobCredentialPublicCertificate=
 StateSaveLocation=/var/spool/slurm.state
 SlurmdSpoolDir=/var/spool/slurmd
 SwitchType=switch/none
-MpiDefault=none
 SlurmctldPidFile=/var/run/slurmctld.pid
 SlurmdPidFile=/var/run/slurmd.pid
-ProctrackType=proctrack/cgroup
-#PluginDir=
+SlurmctldParameters=cloud_dns,idle_on_node_suspend
```

Comment:
I would still keep SlurmctldParameters and CommunicationParameters under `CLOUD CONFIGS OPTIONS` since they are very specific to that and also mentioned in the Cloud Scheduling Guide: https://slurm.schedmd.com/elastic_computing.html